code.js

```diff
@@ -1,8 +1,19 @@
-function flip(array, n) {
-    return array;
+function getBiggest(a, length) {
+    let b = a.slice(0,length)
```

Comment:
Why are you copying the array here?

Reply:
Oops, I forgot we're only supposed to call `flip()` on the array for this exercise. I'll update how I find the largest element without copying it like that.